airflow/providers/amazon/aws/hooks/sagemaker.py

```diff
@@ -786,6 +825,30 @@ def list_training_jobs(
         )
         return results
 
+    def list_processing_jobs(self, **kwargs) -> List[Dict]:  # noqa: D402
+        """
+        This method wraps boto3's list_processing_jobs(). All arguments should be provided via kwargs.
+        Note boto3 expects these in CamelCase format, for example:
+
+        .. code-block:: python
+
+            list_processing_jobs(NameContains="myjob", StatusEquals="Failed")
+
+        .. seealso::
+            https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html#SageMaker.Client.list_processing_jobs
+
+        :param kwargs: (optional) kwargs to boto3's list_training_jobs method
+        :return: results of the list_processing_jobs request
+        """
+
+        config = {}
+        config.update(kwargs)
+        list_processing_jobs_request = partial(self.get_conn().list_processing_jobs, **config)
```

Comment:
```suggestion
        list_processing_jobs_request = partial(self.get_conn().list_processing_jobs, **kwargs)
```
Correct?

Reply:
Good catch, fixed.

Reply:
This should be left as config. We're updating the config with the kwargs and then want to pass that to the list_processing_jobs_request.

Reply:
But why can't we just pass `**kwargs`?

Isn't `**kwargs` equal to `**config` ?

Reply:
Updated.